crds/roman/specs/wfi_area.rmap

```diff
@@ -2,18 +2,18 @@ header = {
     'classes' : ('Match', 'UseAfter'),
     'derived_from' : 'Hand made 2022-02-15',
     'file_ext' : '.asdf',
-    'filekind' : 'PIXELAREA',
-    'filetype' : 'PIXELAREA',
+    'filekind' : 'AREA',
+    'filetype' : 'AREA',
     'instrument' : 'WFI',
-    'ld_tpn' : 'wfi_pixelarea_ld.tpn',
+    'ld_tpn' : 'wfi_area_ld.tpn',
     'mapping' : 'REFERENCE',
-    'name' : 'wfi_pixelarea.rmap',
+    'name' : 'wfi_rmap.rmap',
     'observatory' : 'ROMAN',
     'parkey' : (('ROMAN.META.INSTRUMENT.DETECTOR', 'ROMAN.META.INSTRUMENT.OPTICAL_ELEMENT'), ('ROMAN.META.OBSERVATION.DATE', 'ROMAN.META.OBSERVATION.TIME')),
-    'sha1sum' : '6d600afa3482028ba926ef84fd80da8e806d121a',
-    'suffix' : 'pixelarea',
-    'text_descr' : 'pixelarea',
-    'tpn' : 'wfi_pixelarea.tpn',
+    'sha1sum' : 'b48381f4b3d3a66741eb58556ed2c136431bdb44',
+    'suffix' : 'AREA',
+    'text_descr' : 'AREA',
+    'tpn' : 'wfi_AREA.tpn',
```

Comment:
These should follow the pattern of the other ref types with regard to case.  The suffix is lowercase `area`, the text description (which displays on the website) is capitalized `Area` and the tpn is lowercase `wfi_area.tpn` (this is name of the file below).